src/Nox.Types.EntityFramework/Types/Percentage/PercentageDatabaseConfigurator.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class PercentageDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.Percentage;
+    public bool IsDefault => true;
+
+    public void ConfigureEntityProperty(
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState,
+        EntityTypeBuilder builder,
+        NoxSimpleTypeDefinition property,
+        Entity entity,
+        bool isKey)
+    {
+        var typeOptions = property.PercentageTypeOptions ?? new PercentageTypeOptions();
+
+        builder
+            .Property(property.Name)
+            .IsRequired(property.IsRequired)
+            .HasMaxLength(typeOptions.Digits)
+            .HasConversion<PercentageConverter>();
```

Comment:
Can we update the PercentageConverter to use the Percentage.FromDatabase method when converting db value to value object?

Reply:
PercentageConverter is using `FromDatabase`.